LINK-1970 | update event price section instruction texts

jorilindell opened the pull request:
## Description
Update event price section instruction texts

## Closes
[LINK-1970](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1970)

## Screenshot
<img width="1320" alt="Screenshot 2024-06-05 at 10 45 55" src="https://github.com/City-of-Helsinki/linkedcomponents-ui/assets/24706814/8333440f-b829-4233-aba9-025cbbdef4ee">

[LINK-1970]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1970?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

terovirtanen commented:
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr406-ui.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## e2e tests result is __success__ for https://linkedcomponents-pr406-ui.dev.hel.ninja :laughing::tada::tada::tada:

harriris-vincit commented on src/domain/app/i18n/en.json:
```diff
@@ -610,15 +610,20 @@
         "volunteering": "If the volunteering has multiple prices, click “Add other pricing information” and enter the name of each price group (e.g. Children) in the “Additional price information” field."
       },
+      "infoTextOffers3": {
+        "course": "Jos luot kurssille Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```
```suggestion
        "course": "Jos luot kurssille Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```

harriris-vincit commented on src/domain/app/i18n/en.json:
```diff
@@ -610,15 +610,20 @@
       },
+      "infoTextOffers3": {
+        "course": "Jos luot kurssille Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
+        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```
```suggestion
        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```

harriris-vincit commented on src/domain/app/i18n/en.json:
```diff
@@ -610,15 +610,20 @@
+      "infoTextOffers3": {
+        "course": "Jos luot kurssille Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
+        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
+        "volunteering": "Jos luot vapaaehtoistehtävälle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle."
```
```suggestion
        "volunteering": "Jos luot vapaaehtoistehtävälle Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle."
```

harriris-vincit commented on src/domain/app/i18n/fi.json:
```diff
@@ -610,15 +610,20 @@
       },
+      "infoTextOffers3": {
+        "course": "Jos luot kurssille Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
+        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```
```suggestion
        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```

harriris-vincit commented on src/domain/app/i18n/fi.json:
```diff
@@ -610,15 +610,20 @@
+      "infoTextOffers3": {
+        "course": "Jos luot kurssille Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
+        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
+        "volunteering": "Jos luot vapaaehtoistehtävälle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle."
```
```suggestion
        "volunteering": "Jos luot vapaaehtoistehtävälle Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle."
```

harriris-vincit commented on src/domain/app/i18n/sv.json:
```diff
@@ -610,15 +610,20 @@
       },
+      "infoTextOffers3": {
+        "course": "Om du skapar en Linked Events-registrering för en kurs anger du kundgrupperna och deras priser. Denna information kommer att överföras till registreringen.",
+        "general": "Om du skapar en Linked Events-registrering för ett evenemang anger du kundgrupperna och deras priser. Denna information kommer att överföras till registreringen.",
```
```suggestion
        "general": "Om du skapar en Linked Events-registrering för ett evenemang, ange kundgrupper och deras priser. Denna information kommer att överföras till registreringen.",
```

harriris-vincit commented on src/domain/app/i18n/sv.json:
```diff
@@ -610,15 +610,20 @@
+      "infoTextOffers3": {
+        "course": "Om du skapar en Linked Events-registrering för en kurs anger du kundgrupperna och deras priser. Denna information kommer att överföras till registreringen.",
+        "general": "Om du skapar en Linked Events-registrering för ett evenemang anger du kundgrupperna och deras priser. Denna information kommer att överföras till registreringen.",
+        "volunteering": "Om du skapar en Linked Events-registrering för ett volontärarbete anger du kundgrupperna och deras priser. Denna information kommer att överföras till registreringen."
```
```suggestion
        "volunteering": "Om du skapar en Linked Events-registrering för ett volontärarbete, ange kundgrupper och deras priser.. Denna information kommer att överföras till registreringen."
```

jorilindell commented:
@harriris-vincit I implemented all the translations improvement suggestion. At same time I noticed that there was some Finnish texts in the English translation file. I fixed them as well.

terovirtanen commented:
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr406-ui.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## e2e tests result is __success__ for https://linkedcomponents-pr406-ui.dev.hel.ninja :laughing::tada::tada::tada:
